spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/features/AnalyticsConfiguration.java

```diff
@@ -52,4 +56,24 @@ public FieldValueCounterRepository fieldValueCounterReader(RedisConnectionFactor
 	public AggregateCounterRepository aggregateCounterReader(RedisConnectionFactory redisConnectionFactory) {
 		return new RedisAggregateCounterRepository(redisConnectionFactory, new RetryTemplate());
 	}
+
+	@Bean
```

Comment:
I don't understand what these changes do as it looks to be exactly same as what boot auto-config would do?

Reply:
Right, if only reason to reintroduce these here is to assume that CF server will disable redis auto-config, it kinda pointless. Then these should be added to CF server as just looking dataflow core these are really pointless here.